src/ssr-template-helper.js

```diff
@@ -106,7 +106,7 @@ export class SsrTemplateHelper {
     if (this.isEnabled()) {
       userAssert(
         typeof data['html'] === 'string',
-        'Server side html response must be defined'
+        'Skipping template rendering due to failed fetch'
```

Comment:
Side-nit: This maybe should instead be `user().assertString(data['html'], '...');`

Reply:
Thanks - since that's unrelated to this PR I will leave that out of the current PR but feel free to change that in a separate PR or track it separately!